x-pack/plugins/cloud_security_posture/public/common/navigation/types.ts

```diff
@@ -5,9 +5,7 @@
  * 2.0.
  */
 
-import { CSP_FINDINGS_PATH, CSP_DASHBOARD_PATH } from '../common/constants';
-
-export const navigationLinks = [
-  { name: 'Dashboard', path: CSP_DASHBOARD_PATH },
-  { name: 'Findings', path: CSP_FINDINGS_PATH },
-] as const;
+export interface CspNavigationItem {
+  name: string;
+  path: string;
```

Comment:
not critical, but consider: 

```suggestion
  path: RouteProps["path"];
```

as `path` is going to `Route`

(although that type would be string | string[] so we might need to tweak it more with Extract or casting later on)